src/app/pages/home/components/slider/slider.component.ts

```diff
@@ -0,0 +1,38 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Route, ProductCategory } from '@shared-module';
+
+@Component({
+  selector: 'app-slider',
+  templateUrl: './slider.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SliderComponent {
+  protected readonly route = Route;
+  protected readonly sliders = [
+    {
+      class: 'w-ful h-192 m-auto bg-gradient-to-l from-turquoise-200 to-white relative',
```

Comment:
w-full and h-192 are duplicates, let's move them to template

Reply:
I moved the duplicates to the template, but where should be the 'bg-gradient-to-l'? 
Is the current version ok, or it should be in the ts file to be more readable?